config/config.go

```diff
@@ -143,7 +144,7 @@ func (c *Config) UpdateTempStoragePath() {
 
 func encodeDefTempStorageDir(port, statusPort uint) string {
 	dirName := base64.URLEncoding.EncodeToString([]byte(fmt.Sprintf("%v/%v", port, statusPort)))
```

Comment:
It's better to add host here, because different tidb can listen to addresses on the same port but different hostnames(like 127.0.0.1:4000/10080 and 172.16.5.1:4000/10080).
```suggestion
	dirName := base64.URLEncoding.EncodeToString([]byte(fmt.Sprintf("%v:%v/%v:%v", host, port, statusHost, statusPort)))
```